src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -340,13 +850,148 @@ internal static unsafe float SumAbsU(Span<float> src)
                     pSrcCurrent += 4;
                 }
 
-                result = VectorSum(in result);
+                result = VectorSum(result);
```

Comment:
It may be nice to keep the `in` here and below.  It is good documentation for the reader.

Reply:
It is not just good documentation, it can result in different code being emitted for certain cases.

You should always specify `in` for a variable that is passed that way, IMO.